addons/web/static/src/views/pivot/pivot_view.scss

```diff
@@ -1,123 +1,34 @@
+    width: auto;
+    // Inform the user that he may click on a cell to be redirected to a list view of the
+    // items corresponding to the clicked cell
+    &.o_enable_linking .o_pivot_cell_value:not(.o_empty) {
+        @include o-hover-text-color($body-color, $o-main-link-color);
 
-        thead th:not(.o_pivot_header_cell_closed):not(.o_pivot_header_cell_opened):not(.o_pivot_header_cell) {
-            text-align: center;
-        }
-
-        th, td {
-            border-color: gray('200');
-            border-width: $border-width $border-width $border-width $border-width;
-        }
-
-        th {
-            font-weight: $font-weight-normal;  // bootstrap override
-            background-color: gray('100');
-        }
-
-        @mixin o-pivot-header-cell {
-            cursor: pointer;
-            white-space: nowrap;
-            user-select: none;
-            &:hover {
-                background-color: gray('200');
-                color: $headings-color;
-            }
-        }
-
-        .o_pivot_measure_row, .o_pivot_origin_row {
-            @include o-pivot-header-cell;
-        }
-
-        .o_pivot_header_cell {
-            white-space: nowrap;
-            user-select: none;
-        }
-
-        .o_pivot_header_cell_closed {
-            @include o-pivot-header-cell;
-            &::after, &::before {
-                font-family: FontAwesome;
-                margin-right: 8px;
-            }
-            &::after {
-                content: #{"/*!rtl:''*/"} none;
-            }
-            &::before {
-                content: #{"/*!rtl:none*/"} "";
-            }
-        }
-
-        .o_pivot_header_cell_opened {
-            @include o-pivot-header-cell;
-            &::after, &::before {
-                font-family: FontAwesome;
-                margin-right: 8px;
-            }
-            &::after {
-                content: #{"/*!rtl:''*/"} none;
-            }
-            &::before {
-                content: #{"/*!rtl:none*/"} "";
-            }
-        }
+    }
 
-        .o_pivot_sort_order_asc::after {
-            @include o-caret-up;
-            margin-left: 5px;
-        }
+    // This can be removed after the migration to BS5.
+    th, td {
+        border-width: $border-width;
+    }
 
-        .o_pivot_sort_order_desc::after {
-            @include o-caret-down;
-            margin-left: 5px;
-        }
+    // "user-select-none" class doesn't exist in BS 4.3.1, the user-select property
+    // could be replaced by a class after the migration to BS5
+    .o_pivot_measure_row,
+    .o_pivot_origin_row,
+    .o_pivot_header_cell_closed,
+    .o_pivot_header_cell_opened {
+        user-select: none;
+        @include o-hover-text-color($body-color, $headings-color);
 
-        .o_pivot_cell_value.o_cell_hover {
-            background-color: $table-hover-bg;
+        &:hover {
+            background-color: gray('200') !important;
         }
     }
 
-    .o_pivot_field_selection::after {
```

Comment:
@stefanorigano 
It seems than the `o_pivot_field_selection` has been removed in [this commit](https://github.com/odoo/odoo/commit/36988d0edcfb423ab8ba76f3f675505184641a17#diff-32e8a3944eef15e10cf982ad11168ee11f79ba4aef7c059d8571c6d84ed6cfb9L99) :

I removed the SCSS related to this class.